test/unit/test_auth_webbrowser.py

```diff
@@ -540,7 +540,7 @@ def test_auth_webbrowser_socket_recv_does_not_block_with_env_var(monkeypatch):
     """Authentication by WebBrowser socket.recv Does not block, but retries if BlockingIOError thrown."""
 
     ref_token = "MOCK_TOKEN"
-    rest = _init_rest(REF_SSO_URL, REF_PROOF_KEY)
+    rest = _init_rest(REF_SSO_URL, REF_PROOF_KEY, disable_console_login=True)
```

Comment:
@podung  In a similar way to the rest of the tests, indeed it seems that now the proof_key is encoded when `disable_console_login=False`. For now I just set it to true to make it work, but we can use an if-condition like happens in other tests

Reply:
Sorry for the delays here.  Yep that sounds exactly right to me